_data_weighting.tex

```diff
@@ -163,7 +162,7 @@
 Therefore, an upper bound of 5 may help identify cases that otherwise
 would have convergence issues.  When the parameter hits this upper
 bound, consider either switching those data to a multinomial (which
-behavior the DM is asymptotically approaching) or using a Bayesian prior
+behavior the Dirichlet-multinomial (DM) is asymptotically approaching) or using a Bayesian prior
```

Comment:
There is not an abbreviation listed for Dirichlet-multinomial other places in the text.